src/test_priorityq.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding utf-8 -*-
+import pytest
+
+
+TABLE = [
+    ([(10, 'whatevs'), (5, 'WE')],
+     [(5, 'WE'), (10, 'whatevs')],
+     [(5, 'WE'), (7, 'test'), (10, 'whatevs')]),
+    ([(6, 'whatevs'), (12, 'WE')],
+     [(6, 'whatevs'), (12, 'WE')],
+     [(6, 'whatevs'), (7, 'test'), (12, 'WE')]),
+]
+
+TABLE2 = [
```

Comment:
You guys can do this programmatically rather than manually typing it all out. Look into the `zip` builtin method and `product` in the itertools module.